TickSpec/TickSpec.fs

```diff
@@ -88,6 +88,7 @@ type StepDefinitions (givens,whens,thens,events,valueParsers) =
             xs
             |> Seq.filter (fun m -> m |> isMethodInScope feature scenario)
             |> Seq.map (fun (_,_,_,e) -> e)
+            |> Seq.toList
```

Comment:
to build this into an array, best to use an array comprehension
```
[| for _, _, _, e as x in xs ->
       if isMethodInScope feature scenario x then
           e |]
```

Reply:
Looks good.